### DISCUSSION TITLE
hey just getting started

### DISCUSSION THREAD

**Discussion by Ansh-Rathod** [mindsdb/mindsdb#5395] [Get help]
so I need help running mindsdb locally using anaconda on mac.
i tried to install mindsdb 3 times from hours now using conda but im getting many errors with dependencies and versions.

i created conda env using `conda create -n mindsdb python=3.7`
then `pip3 install mindsdb`
but after the installation `python3 -m mindsdb` it throws many errors.

can any one give the proper version like this eg. mindsdb==22.3.3.0 and python=3.8 to run without any errors.

thanks

**Comment by ZoranPandovski** [comment]
Please use the latest version on PyPi and any Python 3.7.x or 3.8.x version. If you got error please share it with us so we can help you
